### ISSUE TITLE
nopt.js:210 TypeError: Cannot read property 'name' of null

### ISSUE THREAD

**Issue by dseeto** [npm/nopt#56]
Latest release has the following bug:

```
node-sass@3.4.2 install /Users/dseeto/code/node_modules/node-sass
> node scripts/install.js

/Users/dseeto/code/node_modules/nopt/lib/nopt.js:210
    if (t && ((type.name && t.type.name) ?
                   ^
TypeError: Cannot read property 'name' of null
    at validate (/Users/dseeto/code/node_modules/nopt/lib/nopt.js:210:20)
    at validate (/Users/dseeto/code/node_modules/nopt/lib/nopt.js:179:11)
    at /Users/dseeto/code/node_modules/nopt/lib/nopt.js:101:12
    at Array.map (native)
    at /Users/dseeto/code/node_modules/nopt/lib/nopt.js:67:15
    at Array.forEach (native)
    at Function.clean (/Users/dseeto/code/node_modules/nopt/lib/nopt.js:55:21)
    at /Users/dseeto/code/node_modules/npmconf/npmconf.js:411:10
    at Array.forEach (native)
    at validate (/Users/dseeto/code/node_modules/npmconf/npmconf.js:410:11)
```

**Comment by dseeto** [comment]
Looks like there was a commit to fix this. When is the next release?

**Comment by othiym23** [comment]
Closing as a duplicate of #54, and 3.0.6 is now on the registry with the fix, which is 8900f8e3f039d5131c600700994e51f7b312c5fe.
